backend/src/types.ts

```diff
@@ -1170,6 +1171,27 @@ export type TrustyAIKind = K8sResourceCommon & {
   };
 };
 
+export type RoleBindingSubject = {
+  kind: string;
+  apiGroup?: string;
+  name: string;
+};
+
+export type RoleBindingRoleRef = {
+  kind: 'Role' | 'ClusterRole';
+  apiGroup?: string;
+  name: string;
+};
+
+export type RoleBindingKind = K8sResourceCommon & {
+  metadata: {
+    name: string;
+    namespace: string;
+  };
+  subjects: RoleBindingSubject[];
+  roleRef: RoleBindingRoleRef;
+};
```

Comment:
@gitdallas I just realized you should be able to just import `V1RoleBinding`, `V1RoleRef` and `V1Subject` from `'@kubernetes/client-node'` as we do in `backend/src/routes/api/rolebindings/index.ts` instead of creating these types